packages/config-helper/project.json

```diff
@@ -32,14 +32,15 @@
 		"bump-version": {
```

Comment:
As this isn't bumping the version only anymore, please rename the target to `release-new-version` or something similar to reflect what will happen

Reply:
Sure 😄